lib/web_ui/lib/src/engine/browser_detection.dart

```diff
@@ -204,6 +206,34 @@ bool get isIOS15 {
       domWindow.navigator.userAgent.contains('OS 15_');
 }
 
+/// Detect if running on Chrome version 110 or older on Windows.
+///
+/// These versions of Chrome have a bug on Windows which causes
+/// rendering to be flipped upside down.
```

Comment:
Maybe leave a TODO pointing to a Github issue to clean this up later?

Reply:
Done.